wire/cache_internal_test.go

```diff
@@ -15,14 +15,25 @@
 package wire
 
 import (
+	"math/rand"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
+	wtest "perun.network/go-perun/wallet/test"
 	"polycry.pt/poly-go/test"
 )
 
+// NewRandomEnvelope - copy from wire/test for internal tests.
+func NewRandomEnvelope(rng *rand.Rand, m Msg) *Envelope {
```

Comment:
If that function is only used for internal testing, it should be unexported.
```suggestion
func newRandomEnvelope(rng *rand.Rand, m Msg) *Envelope {
```

Reply:
Yes, it could be unexported.

Reply:
https://github.com/hyperledger-labs/go-perun/pull/287/commits/b3e6fbc57aa4b13221fcbdb5758e9c61291211ff